refnx/reduce/reduce.py

```diff
@@ -1098,6 +1131,363 @@ def splice_datasets(self, ds):
         return fname
 
 
+def _check_spin_channel(data, channel):
+    """
+    Checks that the data measured corresponds
+    to the correct spin channel
+
+    Parameters
+    ----------
+    data : PlatypusNexus object
+        Polarised spectra for a given spin channel
+    channel : tuple
+        nominal spin channel - (0,0), (0,1), (1,0), or (1,1)
```

Comment:
not clear what (0, 0) is. Is this mm or uu?
Perhaps we should use enumerations here?